src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -495,14 +493,24 @@ transactionPreProcessImpl (
     // If multisign then return multiSignature, else set TxnSignature field.
     if (signingArgs.isMultiSigning ())
     {
-        Serializer s =
-            buildMultiSigningData (*stpTrans, signingArgs.getSigner ());
-        Blob multiSignature = keypair.secretKey.accountPrivateSign(s.getData());
-        signingArgs.moveMultiSignature (std::move (multiSignature));
+        Serializer s = buildMultiSigningData (*stpTrans,
+            signingArgs.getSigner ());
+
+        auto multisig = ripple::sign (
+            keypair.first,
+            keypair.second,
+            s.slice());
+
+        // TEMPORARY
+        Blob b;
+        auto const ptr = multisig.data();
+        for (std::size_t i = 0; i < multisig.size(); i++)
+            b.push_back (ptr[i]);
```

Comment:
`Blob b(ptr, ptr+multisig.size());` or `signingArgs.moveMultiSignature (Blob (ptr, ptr+multisig.size()));` could replace this loop